movement/io/load_poses.py

```diff
@@ -284,16 +301,101 @@ def _load_from_sleap_labels_file(
     )
 
 
+def _sleap_labels_contains_user_labels_only(labels: Labels) -> bool:
+    """Check if a SLEAP `Labels` object contains only user-labeled instances.
+
+    Parameters
+    ----------
+    labels : Labels
+        A SLEAP `Labels` object.
+
+    Returns
+    -------
+    bool
+        A boolean value indicating whether the labels contain only
+        user-labeled instances.
+    """
+    all_instances = [
+        instance for lf in labels.labeled_frames for instance in lf.instances
+    ]
+    return all(
+        not isinstance(instance, PredictedInstance)
+        for instance in all_instances
+    )
+
+
+def _sleap_user_labels_to_numpy(labels: Labels) -> np.ndarray:
+    """Convert a SLEAP `Labels` object to a NumPy array containing
+    pose tracks with point scores (i.e. confidence) set as 1.0.
+
+    Parameters
+    ----------
+    labels : Labels
+        A SLEAP `Labels` object.
+
+    Returns
+    -------
+    numpy.ndarray
+        A NumPy array containing pose tracks and confidence scores.
+
+    Notes
+    -----
+        This function only considers SLEAP instances in the first
+        video of the SLEAP `Labels` object. It is primarily meant
+        to be used with `Labels` containing only user-labeled instances.
+        If `Labels` contains predicted instances only, this function
+        will overwrite all point scores to 1.0. If `Labels` contains
+        both user-labeled and predicted instances, the returned NumPy
+        array will contain the last occurrence of each tracked instance
+        in each frame.
+
+        This function is adapted from `Labels.numpy()` from the
+        `sleap_io` package [1]_ to allow user-labeled instances.
```

Comment:
There is an extra level of indentation for some reason
```suggestion
    This function only considers SLEAP instances in the first
    video of the SLEAP `Labels` object. It is primarily meant
    to be used with `Labels` containing only user-labeled instances.
    If `Labels` contains predicted instances only, this function
    will overwrite all point scores to 1.0. If `Labels` contains
    both user-labeled and predicted instances, the returned NumPy
    array will contain the last occurrence of each tracked instance
    in each frame.

    This function is adapted from `Labels.numpy()` from the
    `sleap_io` package [1]_ to allow user-labeled instances.
```